client/components/place/place-list/place-list.component.js

```diff
@@ -81,7 +81,7 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         // -----END ADD Place-----
 
         // ----START---- Function for add and remove html classes
-        ctrl.classAddRemove = function(tag, tagClass, add) {
+        ctrl.toggleClass = function(tag, tagClass, add) {
```

Comment:
You don't event need to create your own function, use `[.toggleClass](http://api.jquery.com/toggleClass/)`